### DISCUSSION TITLE
规则添加与更新规划

### DISCUSSION THREAD

**Discussion by manuones** [Tencent/CodeAnalysis#156] [Q&A] [answered]
1、后续如果有新的官方规则，该如何添加进已经搭建起来并且已经运行了一段时间的CodeAnalysis服务中？
2、是否支持自定义规则？
3、相比于Sonarqube，规则的配置自定义是亮点，但是规则的describition目前并不支持。

**Comment by nickcdon** [comment, marked as answer]
@manuones 
1. 后期有新的官方规则包出来后，可更新代码，执行脚本将其入库。会有相应更新文档。
2. 目前已经在计划开放工具管理能力，支持自定义规则
3. 是指自定义规则描述吗？在开放工具管理后，用户可自行修改规则描述。
